components/Animate/AnimateInView.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { useRef } from "react";
+import { useInView, motion, useReducedMotion } from "motion/react";
+import { AnimationMap, type AnimationType } from "./AnimationMap";
+
+export type AnimateInViewProps = {
+  animation?: AnimationType;
+  once?: boolean;
+  duration?: number;
+  delay?: number;
+  children: React.ReactNode;
+  className?: string;
+};
+
+export const AnimateInView = ({
+  animation = "zoomIn",
+  once = true,
+  duration = 0.6,
+  delay,
+  children,
+  className,
+  ...props
+}: AnimateInViewProps) => {
+  const ref = useRef<HTMLDivElement | null>(null);
+  const prefersReducedMotion = useReducedMotion();
+  const isInView = useInView(ref as React.RefObject<Element>, { once });
+
+  // Don't animate if the user has "reduced motion" enabled
+  if (animation === "none") {
+    return <div>{children}</div>;
+  }
+
+  return (
+    <motion.div
+      ref={ref}
+      variants={AnimationMap[animation]}
+      transition={{
+        delay: prefersReducedMotion ? 0 : delay,
+        duration: prefersReducedMotion ? 0 : duration,
+        ease: "easeOut",
+      }}
+      initial="hidden"
+      animate={isInView ? "visible" : "hidden"}
+      className={className}
+      {...props}
+    >
+      {children}
+    </motion.div>
+  );
+};
```

Comment:
Is this using the updated version? 👀 
https://github.com/SU-SWS/ood-giving-campaign/blob/dev/components/Animate/AnimateInView.tsx

Also, is there a reason to use delay and duration 0 for reduced motion? I personally feel that it could be nice to have non zero duration for opacity change for example. LMK your thoughts @rebeccahongsf